### DISCUSSION TITLE
http.send() GET request not wokring ?

### DISCUSSION THREAD

**Discussion by gokulgseven** [pocketbase/pocketbase#4299] [Q&A] [answered]
this is the GET api:  https://app.messageautosender.com/api/v1/message/create?receiverMobileNo=xxxx&message=hi&channelId=5411&api_key=xxxx

This works in postman, and also works directly in browser search bar: but doesnt work in pocketbase 
i had to hide the api_key and the mobile number , but you still get (HTTP Status 401 - Full authentication is required to access this resource) in postman and browser.  Which you dont get in pocketbase. 

expected result with correct key 

```
{
    "status": 200,
    "message": "success",
    "result": {
        "id": 3335894960797927289,
        "groupId": 3335894960797927289,
        "creationTime": "2024-02-05T16:17:56.205+0000"
    }
}
```

expected result with incorrect key: `HTTP Status 401 - Full authentication is required to access this resource`

result in pocketbase: `actual html content of page https://app.messageautosender.com `

what i am doing in pocketbase : 

```
const res = $http.send({
        url: "https://app.messageautosender.com/api/v1/message/create?receiverMobileNo=xxxx&message=hi&channelId=5411&api_key=xxx",
        method: "GET",
        timeout: 120, 
    })
```

**Comment by ganigeorgiev** [comment, marked as answer]
The only thing that would be different from sending the request with `$http.send` and accessing it directly in the browser is that we add a default `Content-Type: application/json` header if not explicitly set.

If you want to get the plain text version, you can try with:


```js
const res = $http.send({
    url: "https://app.messageautosender.com/api/v1/message/create?receiverMobileNo=xxxx&message=hi&channelId=5411&api_key=xxx",
    method: "GET",
    headers: {
        "content-type": "text/html",
    }
})

console.log(res.raw) // HTTP Status 401 - Full authentication is required to access this resource
```

**Reply by gokulgseven** [reply]
working perfectly with the header : 
headers: {
        "content-type": "text/html",
    }
    
   thank you so much for the quick solve
